env-defaults

```diff
@@ -351,9 +351,12 @@ PROMETHEUS_RETENTION_TIME=180d
 # pod may be evicted due to running out of ephemeral space.
 PROMETHEUS_RETENTION_SIZE=1750MB
 
-# how long before the specified alert is triggered/firing
+# how many consecutive minutes to sample in the PromQL query for KubernetesPodNotHealthy
```

Comment:
can you add something in the comments here about what units these are. I still get confused by wheat 10m means.